guides/micronaut-jms-aws-sqs/groovy/src/main/groovy/example/micronaut/DemoConsumer.groovy

```diff
@@ -0,0 +1,19 @@
+package example.micronaut
+
+import io.micronaut.jms.annotations.JMSListener
+import io.micronaut.jms.annotations.Queue
+import io.micronaut.messaging.annotation.MessageBody
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+import static io.micronaut.jms.sqs.configuration.SqsConfiguration.CONNECTION_FACTORY_BEAN_NAME
+
+@JMSListener(CONNECTION_FACTORY_BEAN_NAME)
```

Comment:
If possible all Groovy classes should be annotated with `@CompileStatic`. I've had problems using this with controllers, but generally it works and will generate more efficient bytecode